events-to-histogram/src/main.rs

```diff
@@ -0,0 +1,154 @@
+mod metrics;
```

Comment:
Why has this file been added?

Reply:
I think I must have messed up the merge. It's hard to tell what is no longer needed, I'll usually opt for keeping instead of discarding changes.